_drafts/oryx/ssr.md

```diff
@@ -0,0 +1,137 @@
+
+
+## SSR Implementation
+
+### Traditional server-based SSR
+
+Traditional SSR implementation involves rendering the initial HTML content on a server, typically powered by Node.js. Oryx supports this scenario for both development and production builds with `createServer` utility in `@spryker-oryx/application/server` package:
+
+```ts
+  import { createServer } from '@spryker-oryx/application/server';
+
+  createServer(config).run();
+```
+
+`createServer` utility is using ExpressJS server under the hood and provides a convenient way to configure and run the server and also exposes the ExpressJS server instance for further customization.
+
+### Serverless SSR using Lambda
+
+Serverless SSR, on the other hand, employs on-demand serverless platforms such as AWS Lambda for HTML rendering, eliminating the need for a dedicated server. 
+
+Oryx supports this scenario for both development and production builds with `storefrontHandler` utility in `@spryker-oryx/application/lambda'` package:
+
+```ts
+import { storefrontHandler } from '@spryker-oryx/application/lambda';
+
+return (event, context) => storefrontHandler(event, {
+    ...context,
+    root: 'file:///apps/storefront/functions/ssr/index.js',
+    index: '../../client/index.html',
+    entry: '../../server/render.js',
+    component: '<root-app></root-app>',
+});
+```
+
+### Caching, CDN
+
+Caching and CDNs are additional layers that can significantly enhance the performance of SSR applications by reducing server load and accelerating content delivery.
+
+**Caching**: Caching acts as a layer on top of rendering. Once the HTML content is rendered, it's cached to swiftly serve repeated requests without the need for re-rendering. Various caching solutions could be used, such as Varnish, Redis, or even service-specific solutions such as Cloudflare's caching services.
+
+**Content Delivery Networks (CDNs)**: CDNs distribute cached content across a network of servers located worldwide. This ensures that users receive content from the nearest server, which significantly improves delivery speed and reduces latency.
+
+In the context of serverless platforms like Netlify, you can leverage Netlify's on-demand builder.
+It works as a simple wrapper to SSR Lambda handler and automatically feed Netlify Edge CDN by building and caching a page when it's requested, thereby reducing the load on the serverless function and improving performance.
+
+```mermaid
+sequenceDiagram
+  participant User
+  participant CDN
+  participant SSR Service
+  participant Backend
+  
+  User->>CDN: Request page
+    CDN->>SSR Service: Request SSR page
+    SSR Service-->>CDN: Cache SSR page
+  CDN-->>User: Deliver SSR page
+  User->>CDN: Request static assets (CSR)
+  CDN-->>User: Deliver static JS assets
+  Note over User: Hydration occurs, SPA mode
+  User->>Backend: CSR Interacts with backend app
+```
+
+## SSR in Oryx
```

Comment:
I'd drop this section, move some pieces to the introduction doc and some to the associated developer doc. 
- The "configuration" and "building with SSR support" sections can be a nice intro to the dev doc
- The hydration could move to the "introduction doc" (next to caching)
- 
-